lib/helpers.js

```diff
@@ -31,21 +35,21 @@ exports.reportValidationError = function(err, logFn) {
  */
 exports.validateAppName = function(name) {
   if (name.charAt(0) === '.') {
-    return 'Application name cannot start with .: ' + name;
+    return g.f('Application name cannot start with .: %s', name);
   }
   if (name.match(/[\/@\s\+%:]/)) {
-    return 'Application name cannot contain special characters (/@+%: ): ' +
-      name;
+    return g.f('Application name cannot contain special characters ' +
+      '(/@+%: ): %s', name);
   }
   if (name.toLowerCase() === 'node_modules') {
-    return 'Application name cannot be node_modules';
+    return g.f('Application name cannot be node_modules');
   }
   if (name.toLowerCase() === 'favicon.ico') {
-    return 'Application name cannot be favicon.ico';
+    return g.f('Application name cannot be favicon.ico');
   }
   if (name !== encodeURIComponent(name)) {
-    return 'Application name cannot contain special characters escaped by ' +
-      'encodeURIComponent: ' + name;
+    return g.f('Application name cannot contain special characters escaped by' +
+    ' %s encodeURIComponent: ', name);
```

Comment:
I think `%s` should after encodeURIComponent?
should be

g.f('Application name cannot contain special characters escaped by' +
  ' encodeURIComponent:  %s', name)